src/app/content/components/Page.spec.tsx

```diff
@@ -61,6 +62,32 @@ const makeEvent = (doc: Document) => {
   return event;
 };
 
+const references: Array<PageReferenceMap | PageReferenceMapError> = [
+  {
+    match: '/content/link',
+    params: {
+      book: {
+        slug: 'book-slug-1',
+      } ,
+      page: {
+        slug: 'page-title',
+      },
+    },
+    state: {
+      bookUid: 'book',
+      bookVersion: 'version',
+      pageUid: 'page',
+    },
+  },
+  {
+    reference: {
```

Comment:
can you change the error format to be more consistent with the regular format? otherwise you get stilly stuff like `reference.reference`, i feel like the error should just have keys `match` and `type` right?